README.md

```diff
@@ -100,3 +100,13 @@ Run the autoformatter on your code:
 ```bash
 yarn fix-format
 ```
+
+## About
+
+Calder was named in honor of Alexander Calder, the American sculptor who is best
+known for using wire to construct three-dimensional abstract line drawings of
```

Comment:
maybe we should change "best known" to "known for"? just because I dont feel confident speaking on behalf of the art world for what his MOST significant contribution is :')

Reply:
Good call fam, I agree! 💯